### PR TITLE
Import from ethereum events sheet

### PR DESCRIPTION
**Pull request by wslyvh**
Adding an importer to fetch community events from the Ethereum Events sheet.
https://docs.google.com/spreadsheets/d/1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ/

## Description

It's hard to maintain an up to date overview of community events. This PR adds an importer from a known Google Sheet to help populate the community events page. The idea is that you can add multiple importers, while also keeping your own, internal JSON file where people can suggest to add their own events, so you're not just relying on the sheet for populating the events overview.

## TODOs

- [x] Try to match a write events to events JSON on the repo
- [x] Add a Github Action periodically run the importer as a cron job

## Summary by CodeRabbit

- **New Features**
	- Enhanced the functionality to import community events data from Ethereum into the application.
	- Introduced a GitHub Actions workflow for importing community events weekly.

### CODE REVIEW

**Comment by wslyvh** [comment]
Ready for review

**Comment by nloureiro** [comment]
This looks great from my perspective. 

I will follow up with another PR with the design change, but we can push this to production from my side.

@pettinarip or @wackerow : I might need some of your eyes here.

Thank you, @wslyvh, for this!!!!

**Review by pettinarip — APPROVED** [review]
LGTM @wslyvh 💪🏼 nice job!

File: src/scripts/events/ethereum-events-import.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    return {}
+  }
+}
+
+export async function EthereumEventsImport() {
+  const googleApiKey = process.env.GOOGLE_API_KEY
+  const sheetId = "1NEu_FCc1hnGAuRgPmbXXpf0h2lCrCOlMKbbFEqgkVDQ"
+
+  if (!googleApiKey || !sheetId) {
+    console.warn("googleApiKey or sheetId not set")
```
Comment (wackerow) [inline src/scripts/events/ethereum-events-import.ts:37]: We're hard-coding `sheetId`, thus will always be available:

```suggestion
  if (!googleApiKey) {
    console.warn("googleApiKey not set")
```

**Review by wackerow — APPROVED** [review]
Nice! Thanks @wslyvh, this looks great! Just gonna push one small commit to remove the guard for a hard-coded variable and then pull this in. Thanks again!

(On the side, we need to double-check that our GOOGLE_API_KEY token has the appropriate permissions and update if not.)

**Comment by wackerow** [comment]
@all-contributors please add @wslyvh for infra

File: src/scripts/events/ethereum-events-import.ts (hunk 1 above)
Comment (wslyvh) [reply src/scripts/events/ethereum-events-import.ts]: Good catch!

**Pull request merged**